aspnetcore/mvc/views/razor.md

```diff
@@ -256,6 +256,36 @@ Without the `@:` in the code, a Razor runtime error is generated.
 
 Extra `@` characters in a Razor file can cause compiler errors at statements later in the block. These compiler errors can be difficult to understand because the actual error occurs before the reported error. This error is common after combining multiple implicit/explicit expressions into a single code block.
```

Comment:
```suggestion
Extra `@` characters in a Razor file can cause compiler errors at statements later in the block. These extra `@` compiler errors:

* Can be difficult to understand because the actual error occurs before the reported error.
* Is common after combining multiple implicit and explicit expressions into a single code block.
```